public class Test041 {
  // Initialize the sample data.
  int customerAccountTransactio = 1;

  // Process the current value.
  int customerAccountTransactionProcessingConfigurationReferenceIdentifierRecordContextProvi = 1234;

  // Record the operation result.
  int customerAccountTransactionProcessingConfigurationReferenceIdentifierRecordContextProviderFactoryRegi = 3;

}
